assets/icons/icon-dataset-el.svg

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   width="24"
+   height="24"
+   viewBox="0 0 24 24"
+   version="1.1"
+   id="svg841"
+   sodipodi:docname="icon-dataset-el-v3.svg"
+   inkscape:version="0.92.4 (5da689c313, 2019-01-14)">
+  <metadata
+     id="metadata847">
+    <rdf:RDF>
+      <cc:Work
+         rdf:about="">
+        <dc:format>image/svg+xml</dc:format>
+        <dc:type
+           rdf:resource="http://purl.org/dc/dcmitype/StillImage" />
+        <dc:title />
+      </cc:Work>
+    </rdf:RDF>
+  </metadata>
+  <defs
+     id="defs845" />
+  <sodipodi:namedview
+     pagecolor="#ffffff"
+     bordercolor="#666666"
+     borderopacity="1"
+     objecttolerance="10"
+     gridtolerance="10"
+     guidetolerance="10"
+     inkscape:pageopacity="0"
+     inkscape:pageshadow="2"
+     inkscape:window-width="2560"
+     inkscape:window-height="1377"
+     id="namedview843"
+     showgrid="false"
+     inkscape:zoom="50.416667"
+     inkscape:cx="12"
+     inkscape:cy="14.380165"
+     inkscape:window-x="-8"
+     inkscape:window-y="-8"
+     inkscape:window-maximized="1"
+     inkscape:current-layer="svg841" />
+  <path
+     d="m 1.5322315,18.033057 c -0.77999134,1.881107 1.6616426,2.77762 2.4008263,1.194182 1.5145123,-3.48281 4.1667321,-10.1934181 4.4767266,-10.9085405 0.3642038,-0.7594805 1.0393882,-0.751551 1.313356,-0.046001 0.2739679,0.70555 1.8999766,4.4881685 2.5756726,5.7914955 0.675697,1.303327 1.785363,1.278289 2.420196,-0.02863 0.634833,-1.306943 0.683848,-1.474463 0.937971,-2.060311 0.254123,-0.585848 0.882985,-0.612912 1.132598,-0.06014 0.259326,0.574274 2.878308,6.394771 3.368458,7.362405 0.746436,1.473582 3.028667,0.464443 2.398577,-1.087815 C 21.926522,16.637439 17.709897,7.206987 17.335123,6.1731434 16.96035,5.1392999 15.753419,5.1860949 15.369113,6.1956881 14.984806,7.2052814 14.267355,8.8794611 14.032439,9.3766552 13.797523,9.8738493 13.267972,10.006537 13.039962,9.4560088 12.317247,7.7110185 10.720401,3.7233017 10.168711,2.2545505 9.7823247,1.2983404 8.5276646,1.2780792 8.0919421,2.38533 7.5627449,3.7501867 2.3414642,16.139571 1.5322315,18.033057 Z"
+     id="path839"
+     inkscape:connector-curvature="0"
+     sodipodi:nodetypes="ccczzzzsszzzzsccc" />
+</svg>
```

Comment:
These 50+ lines can be replaced with:
```
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
  <path d="M.3 20.1c-.9 2.1 1.8 3.1 2.7 1.3L8 9.2c.4-.8 1.2-.8 1.5-.1s2.1 5 2.9 6.4c.8 1.5 2 1.4 2.7 0l1-2.3c.3-.7 1-.7 1.3-.1l3.8 8.2c.8 1.6 3.4.5 2.7-1.2-.9-1.6-5.6-12.1-6-13.2s-1.8-1.1-2.2 0a33 33 0 01-1.5 3.5c-.3.6-.9.7-1.1.1l-3.2-8c-.4-1.1-1.8-1.1-2.3.1L.3 20.1z"/>
</svg>
```
And it will have the same icon as a result.

Reply:
Have modified the icon accordingly.